packages/now-cli/src/util/input/input-project.ts

```diff
@@ -88,6 +89,7 @@ export default async function inputProject(
       type: 'input',
       name: 'newProjectName',
       message: `What’s your project’s name?`,
+      default: !detectedProject ? detectedProjectName : undefined,
```

Comment:
```suggestion
      default: detectedProject ? detectedProject.name : detectedProjectName,
```
I'm not sure but I think this should be changed
I'm not sure but I think you

Reply:
It means that we only prefill when the inferred project name (--name, now.json, folder name) is not an existing project.